back/src/test/java/com/baba/back/oauth/service/OAuthServiceTest.java

```diff
@@ -84,7 +84,32 @@ class OAuthServiceTest {
         given(memberRepository.findById(멤버1.getId())).willReturn(Optional.of(멤버1));
         given(accessTokenProvider.createToken(멤버1.getId())).willReturn(accessToken);
         given(refreshTokenProvider.createToken(멤버1.getId())).willReturn(refreshToken);
-        given(tokenRepository.save(any(Token.class))).willReturn(any());
+        given(tokenRepository.findByMember(멤버1)).willReturn(Optional.of(new Token(멤버1, refreshToken)));
+
+        // when
+        final SocialLoginResponse response = oAuthService.signInKakao(new SocialTokenRequest(validToken));
+
+        // then
+        assertAll(
+                () -> assertThat(response.httpStatus()).isEqualTo(HttpStatus.OK),
+                () -> assertThat(response.tokenResponse()).isEqualTo(new LoginTokenResponse(accessToken, refreshToken))
+        );
+
+        then(tokenRepository).should(times(1)).save(any());
+    }
+
+    @Test
+    void 소셜_로그인시_가입이_되어_있고_저장된_refresh토큰이_없어도_access토큰과_refresh토큰을_발급한다() {
+        // given
+        final String validToken = "validToken";
+        final String accessToken = "accessToken";
+        final String refreshToken = "refreshToken";
+
+        given(oAuthClient.getMemberId(any())).willReturn(멤버1.getId());
```

Comment:
```suggestion
        given(oAuthClient.getMemberId(validToken)).willReturn(멤버1.getId());
```
다시 보니까 `any()` 보다는 `validToken`으로 명시해주는게 좋아보이네요

Reply:
좋습니다 놓쳤네요! 
수정했습니다.